docs/changelog-fragments.d/866.bugfix.md

```diff
@@ -0,0 +1,5 @@
+Detect and error for missing `/dev/mqueue` directory when
```

Comment:
```suggestion
Started erroring out loudly on missing `/dev/mqueue/` directory when
```
(past tense for consistency —see the RTD preview + trailing slash to emphasize that it's a dir)

Reply:
rephrased. lmk

Reply:
Looks good, resolving.